### ISSUE TITLE
Occurrences marking should support case sensitive and insensitive modes

### ISSUE THREAD

**Issue by DPovilas** [bobbylight/RSyntaxTextArea#374]
I think RSyntaxtTextArea is missing a parameter where we could specify if occurrences marking should be case sensitive or insensitive. I think it misses this functionality, am I right?

**Comment by bobbylight** [comment]
Interesting, I never thought about it.  How would this be configured?  Is it programmatic only?  I guess a UI concern should be registered in the sister `RSTAUI` project, and you're just asking for a programmatic way to toggle case sensitivity?

**Comment by DPovilas** [comment]
Yes, it would be configured only programmatically. The behavior should be the same as it is in Notepad++. There you just double click a word and all the occurrences are marked despite the case of the letters. Somebody may need the current behavior, but somebody would prefer to mark occurrences with case insensitive.
